core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -176,6 +176,12 @@
       <trans-unit id="tl_page.cache.1">
         <source>The number of seconds after which the page should no longer be considered fresh by shared caches.</source>
       </trans-unit>
+      <trans-unit id="tl_page.forceCache.0">
+        <source>Force server cache for logged-in members</source>
+      </trans-unit>
+      <trans-unit id="tl_page.forceCache.1">
+        <source>By default, the server cache is disabled for logged-in members. You can force caching with this checkbox (ATTENTION: This can lead to information leak between logged in members, only do this if you know what you're doing!).</source>
```

Comment:
> ATTENTION: This can lead to information leak between logged in members

IMO this is not correct. We only store responses in the cache if no cookies are involved regardless of the `forceCache` setting.

The only thing that can happen if `forceCache` gets used incorrectly is that a logged in member gets the public response from the cache instead of a private one personalized to him.

Reply:
True, fixed in e4c289aac71ca8875703b01891eb33bc4eb22376.